### ISSUE TITLE
Batch experiments within ask-tell mode

### ISSUE THREAD

**Issue by latmarat** [dragonfly/dragonfly#60]
Hi, thanks for a great BO framework. 

I have a question similar to Issue #52 in the sense that I want to do a real experiment on points suggested by dragonfly BO. I can see that the new ask-tell interface allows us to do that (following[ this demo](https://github.com/dragonfly/dragonfly/blob/master/examples/detailed_use_cases/in_code_demo_ask_tell.py)). 

My question is how to use this ask-tell interface in a situation of multiple workers? That is, could you please suggest settings to use with optimizers so that the line 

```
x = opt.ask()
```

would return multiple (say 10) points to test at once? Then I would come back with 10 values of y to feed into `opt.tell([(x, y)])` and proceed to the next iteration?

Thank you!

**Comment by kirthevasank** [comment]
You can call opt.ask() 10 times. The procedure is randomized to avoid any pathological situations where the same point is returned 10 times (Please let us know if this is not the case).

**Comment by latmarat** [comment]
Makes sense, thanks! I just discovered that opt.ask() has an optional argument n_points. Is setting n_points=10 identical to running opt_ask() 10 times? 

Also, could you please recommend the best way to feed existing (initial) experiment results into the opt? Is it simply by running opt.tell([(x, y)]) before opt.ask() or by some other function (e.g., perform_initial_queries). If so, what is the correct syntax for perform_initial_queries? 

Thank you!

**Comment by kirthevasank** [comment]
Yes, that is what n_points does. We have updated the example to reflect this: https://github.com/dragonfly/dragonfly/blob/master/examples/detailed_use_cases/in_code_demo_ask_tell.py

For feeding previous results and saving results to a file, look at the following example: https://github.com/dragonfly/dragonfly/blob/master/examples/detailed_use_cases/in_code_demo_single_objective.py
